statsmodels/distributions/copula/copulas.py

```diff
@@ -462,7 +462,7 @@ def fit_corr_param(self, data):
             warnings.warn("currently only first pair of data are used"
                           " to compute kendall's tau")
         tau = stats.kendalltau(x[:, 0], x[:, 1])[0]
-        return self._theta_from_tau(tau)
```

Comment:
This was in the base class but had to implementation. There were some `theta_from_tau` methods in subclasses.